src/Pages/HomePage/Footer.jsx

```diff
@@ -1,224 +0,0 @@
-import React from "react";
-import foot from "../../Styles/Footer.module.css";
-import {
-  FaFacebook,
-  FaInstagram,
-  FaLinkedin,
-  FaTwitter,
-  FaYoutube,
-} from "react-icons/fa";
-
-const Footer = () => {
-  return (
-    <Box className={foot.Container}>
-      <Box style={{ padding: "50px" }}>
-        <Heading
-          as="h4"
-          size="md"
-          style={{
-            paddingLeft: "60px",
-            color: "white",
-          }}
-        >
-          We operate in many other cities
-        </Heading>
-        <br />
-        <Heading
-          as="h4"
-          size="md"
-          style={{
-            paddingLeft: "60px",
-            color: "white",
-          }}
-        >
-          INDIA
-        </Heading>
-        <br />
-
-        <Box className={foot.FooterContent}>
-          <Box>
-            <Text fontSize="sm">Banagalore</Text>
-            <br />
-            <Text fontSize="sm">Chennai</Text>
-            <br />
-            <Text fontSize="sm">Hyderabad</Text>
-            <br />
-            <Text fontSize="sm">Ahemadabad</Text>
-            <br />
-            <Text fontSize="sm">Mangalore</Text>
-            <br />
-            <Text fontSize="sm">Bhopal</Text>
-            <br />
-            <Text fontSize="sm">Madurai</Text>
-            <br />
-            <Text fontSize="sm">Vadodara</Text>
-            <br />
-            <br />
-            <Heading as="h4" size="md">
-              INDONESIA
-            </Heading>
-
-            <br />
-            <Text fontSize="sm">Jakarta</Text>
-          </Box>
-          <Box>
-            <Text fontSize="sm">Bangalore</Text>
-            <br />
-            <Text fontSize="sm">Chennai</Text>
-            <br />
-            <Text fontSize="sm">Hyderabad</Text>
-            <br />
-            <Text fontSize="sm">Ahemadabad</Text>
-            <br />
-            <Text fontSize="sm">Mangalore</Text>
-            <br />
-            <Text fontSize="sm">Bhopal</Text>
-            <br />
-            <Text fontSize="sm">Madurai</Text>
-            <br />
-            <Text fontSize="sm">Vadodara</Text>
-            <br />
-            <br />
-            <Heading as="h4" size="md">
-              VIETNAM
-            </Heading>
-            <br />
-            <Text fontSize="sm">Ho Chi Minh</Text>
-          </Box>
-          <Box>
-            <Text fontSize="sm">Banagalore</Text>
-            <br />
-            <Text fontSize="sm">Chennai</Text>
-            <br />
-            <Text fontSize="sm">Hyderabad</Text>
-            <br />
-            <Text fontSize="sm">Ahemadabad</Text>
-            <br />
-            <Text fontSize="sm">Mangalore</Text>
-            <br />
-            <Text fontSize="sm">Bhopal</Text>
-            <br />
-            <Text fontSize="sm">Madurai</Text>
-            <br />
-            <Text fontSize="sm">Vadodara</Text>
-            <br />
-            <br />
-            <Heading as="h4" size="md">
```

Comment:
Give comments above each section to describe what that section is actually about